ngraph_bridge/enable_variable_ops/ngraph_replace_variable_modifiers.cc

```diff
@@ -35,6 +35,25 @@ namespace ngraph_bridge {
 // 1. ReplaceModifiers
 // 2. RewriteForTracking
 
+Status CreateBinaryOpNode(NodeBuilder::NodeOut* ndef_gen_op,
+                          std::string new_name_op, std::string op_type,
+                          NodeBuilder::NodeOut input1,
+                          NodeBuilder::NodeOut input2, DataType dtype,
+                          Node*& node, Graph*& graph) {
```

Comment:
This node need not be passed the device name can be passed as string. To be addressed in a later PR.